CHANGELOG/CHANGELOG-0.6.md

```diff
@@ -0,0 +1,82 @@
+## v0.5.3
+
+Changes since `v0.5.0`:
+
+### API Change
+
+- A `stopPolicy` field in the ClusterQueue allows to hold or drain a ClusterQueue (#1299, @trasc)
+- Add a lendingLimit field in ClusterQueue's quotas, to allow restricting how much of the unused resources by the ClusterQueue can be borrowed by other ClusterQueues in the cohort.
+  In other words, this allows a quota equal to `nominal-lendingLimit` to be exclusively used by the ClusterQueue. (#1385, @B1F030)
+- Add validation for clusterQueue: when cohort is empty, borrowingLimit must be nil. (#1525, @B1F030)
+- Allow decrease reclaimable pods to 0 for suspended job (#1277, @yaroslava-serdiuk)
+- MultiKueue: Add Path location type for cluster KubeConfigs. (#1640, @trasc)
+- MultiKueue: Add garbage collection of deleted Workloads. (#1643, @trasc)
+- MultiKueue: Multi cluster job dispatching for k8s Job. This doesn't include support for live status updates. (#1313, @trasc)
+- Support for a mechanism to suspend a running Job without requeueing (#1252, @vicentefb)
+- Support for preemption while borrowing (#1397, @mimowo)
+- The leaderElection field in the Configuration API is now defaulted.
+  Leader election is now enabled by default. (#1598, @astefanutti)
+- Visibility API: Add an endpoint that allows a user to fetch information about pending workloads and their position in LocalQueue. (#1365, @PBundyra)
+- Visibility API: Introduce an on-demand API endpoint for fetching pending workloads in a ClusterQueue. (#1251, @PBundyra)
+- Visibility API: extend the information returned for the pending workloads in a ClusterQueue, including the workload position in the queue. (#1362, @PBundyra)
+- WaitForPodsReady: Add a config field to allow admins to configure the timestamp used when sorting workloads that were evicted due to their Pods not becoming ready on time. (#1542, @nstogner)
+- WaitForPodsReady: Support a backoff re-queueing mechanism with configurable limit. (#1709, @tenzen-y)
+
+### Feature
+
+- Add Prebuilt Workload support for JobSets. (#1575, @trasc)
+- Add events for transitions of the provisioning AdmissionCheck (#1271, @stuton)
+- Add prebuilt workload support for batch/job. (#1358, @trasc)
+- Add support for groups of plain Pods. (#1319, @achernevskii)
+- Allow configuring featureGates on helm charts. (#1314, @B1F030)
+- At log level 6, the usage of ClusterQueues and cohorts is included in logs.
+  
```

Comment:
@alculquicondor Can we remove this empty line?

Reply:
It's intentional. This is how it renders:

![image](https://github.com/kubernetes-sigs/kueue/assets/1299064/02595c49-65d5-4b85-9ee4-91be6d48b6b3)